react/src/pages/ResourcesPage.tsx

```diff
@@ -71,10 +70,22 @@ const ResourcesPage: React.FC<ResourcesPageProps> = (props) => {
         </Suspense>
       ) : null}
       {curTabKey === 'resourceGroup' ? (
-        // @ts-ignore
-        <backend-ai-resource-group-list active />
+        <>
+          <Suspense
+            fallback={
+              <Skeleton
+                active
+                style={{ padding: token.paddingContentVerticalLG }}
```

Comment:
It seems that excessive padding has been applied due to this style.

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/XqC2uNFuj0wg8I60sMUh/41222d8d-e724-48a4-a89a-70d260d61b78.png)